src/components/import/Period.jsx

```diff
@@ -38,48 +45,66 @@ const Period = ({ calendar, period, datasetPeriodType, onChange }) => {
     return (
         <div className={styles.container}>
             <h2>{i18n.t('Period')}</h2>
-            <p>
-                {i18n.t(
-                    'Daily values will be imported between start and end dates'
-                )}
-            </p>
-            <div className={styles.pickers}>
-                <PeriodType
-                    periodType={periodType}
-                    datasetPeriodType={datasetPeriodType}
-                    onChange={(periodType) =>
-                        onChange({ ...period, periodType })
-                    }
-                />
-                <CalendarInput
-                    label={i18n.t('Start date')}
-                    date={startTime}
-                    calendar={calendar}
-                    locale={locale || 'en'}
-                    defaultVal={startTime}
-                    onDateSelect={({ calendarDateString }) =>
-                        onChange({ ...period, startTime: calendarDateString })
-                    }
-                />
-                <CalendarInput
-                    label={i18n.t('End date')}
-                    date={endTime}
-                    calendar={calendar}
-                    locale={locale || 'en'}
-                    defaultVal={endTime}
-                    onDateSelect={({ calendarDateString }) =>
-                        onChange({ ...period, endTime: calendarDateString })
-                    }
-                />
-                <TimeZone period={period} onChange={onChange} />
-            </div>
+            {hasNoPeriod ? (
+                <p>
+                    {i18n.t(
+                        "The data don't have a period. It will be imported with the year the data was collected ({{datasetPeriod}}).",
```

Comment:
Or we say something like "the data will be assigned a default yearly period that matches hte year the data was collected" ?

Reply:
Changed to: 

```
i18n.t(
  'The data will be assigned a default yearly period that matches the year it was collected: {{datasetPeriod}}',
  { datasetPeriod }
)
```